cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -202,6 +202,11 @@ set(SYCL_FLAGS "")
 set(SYCL_LINK_FLAGS "")
 list(APPEND SYCL_FLAGS "-fsycl")
 list(APPEND SYCL_LINK_FLAGS "-fsycl")
+# Workaround about XPU ABI neutral build
+if(LINUX)
```

Comment:
Let's do the same for Windows.

Reply:
Seems like Windows doesn't have [dual abi](https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html) issue.